ingestion-beam/src/main/java/com/mozilla/telemetry/transforms/NormalizeAttributes.java

```diff
@@ -61,7 +61,8 @@ public static String normalizeOs(String name) {
       return WINDOWS;
     } else if (name.startsWith("Darwin")) {
       return MAC;
-    } else if (name.contains("Linux") || name.contains("BSD") || name.contains("SunOS")) {
+    } else if (name.contains("Linux") || name.contains("BSD") || name.contains("SunOS")
```

Comment:
FWIW, DragonFly which was forked from FreeBSD ~15 years ago doesn't contain BSD in system name.